src/site/layouts/office.drupal.liquid

```diff
@@ -24,71 +24,89 @@
           <div class="vads-l-grid-container--full--full">
             <h1>{{ title }}</h1>
             <div class="vads-l-grid-container--full--full">
-              {{ fieldDescription }}
-              {% if title == 'Events' %}
+              <div class="va-introtext">
+                <p class="office-show" id="office-main-description">
+                  {{ fieldDescription }}
+                </p>
+                {% if fieldAssetLibraryDescription %}
+                <p class="library-show" id="office-benefits-description">
+                  {{ fieldAssetLibraryDescription }}
+                </p>
+                {% endif %}
+                {% if fieldEventListingDescription %}
+                <p class="events-show" id="office-events-description">
+                  {{ fieldEventListingDescription }}
+                </p>
+                {% endif %}
+              </div>
+              <div class="office-show">
```

Comment:
Perhaps naive question: Would it make more sense for this to be the following?
```
{% if fieldBody.processed %}
  <div class="office-show">
    {{ fieldBody.processed }}
  </div>
{% endif %}
```

Reply:
It sure would - changed, @cvalarida